### PR TITLE
go.mod, nix.flake: update tailscale to 1.76, remove docker from flake

### PR DESCRIPTION
**Pull request by Erisa**

### CODE REVIEW

File: cmd/tclipd/static/css/base.css [hunk 1]
```diff
@@ -1,5 +1,113 @@
+*, ::before, ::after {
```
Comment (kradalby) [inline cmd/tclipd/static/css/base.css:1]: Lets split this commit into a separate PR

Comment (Erisa) [reply cmd/tclipd/static/css/base.css]: Reverted this for now!

**Review by kradalby — APPROVED** [review]
Remember to squash 👍
